pkg/models/diagNode.go

```diff
@@ -0,0 +1,56 @@
+package models
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+type CRNode struct {
+	Name            string
+	GVR             schema.GroupVersionResource
+	GetResourceName func(string) string
+	GetListOptions  func([]string) metav1.ListOptions
+	Leavies         []*CRNode
```

Comment:
What’s this? Explain. The term `Leavies` is interesting but likely not obvious...

Reply:
oh, typo,  it is `leaves` since I am using tree structure to build the hierarchy of ksvc CRDs.